code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -15,7 +15,9 @@
 	interaction_flags_mouse_drop = NEED_HANDS | ALLOW_RESTING
 
 	///The ID currently stored in the computer.
-	var/obj/item/card/id/computer_id_slot
+	var/obj/item/card/id/stored_id
+	///The alt slot, only used by certain UIs like the access app.
+	var/obj/item/card/id/alt_stored_id
```

Comment:
This would be nice if we plan on having more apps that make use of the secondary ID but since it seems like you're going out of your way to limit this to just the HoP's app, should we just make it use ``application_item_interaction`` instead?

Reply:
It's binded to a trait, not hardcoded to only work with the HoP app or anythin